src/Game/Data.elm

```diff
@@ -49,4 +49,5 @@ type Screen
 
 type InventoryMsg source target
     = InventoryMsg
+    | ExitMsg
```

Comment:
I can't really comment on the architecture because I didn't write cotwelm, but this feels like it should be using a Command rather than a message.